### DISCUSSION TITLE
官网提供的pagviewer里的文本编辑功能

### DISCUSSION THREAD

**Discussion by liaohongyuan** [Tencent/libpag#1619] [Q&A]
PAGViewer里文本编辑功能里可以调整文本的字体类型、字号、是否粗体、是否斜体、文本颜色、文本内容等，我想知道调整字体类型、字号、粗图、斜体、颜色这些功能都对应哪些api啊，我咋找不到呢

**Comment by kevingpqi123** [comment]
可以看下 PAGText 的相关接口：https://pag.art/docs/apis-ios.html
